### ISSUE TITLE
Add a `get_predicted_class` method or similar to `Model`

### ISSUE THREAD

**Issue by goodfeli** [tensorflow/cleverhans#847]
The codebase, especially the tests, is full of calls that take the argmax across logits / probs, may as well simplify this.
